lang/pt_BR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -11996,14 +11996,19 @@ msgid "You seem to be having trouble with your upgrade."
 msgstr "Parece que você esta tendo problemas com sua atualização."
 
 #: centreon-web/www/install/step_upgrade/step4.php:70
-msgid "You may refer to the line that causes problem in order to find out more about the issue."
+msgid "You may refer to the line in the specified file in order to correct the issue."
 msgstr "Você deve verificar a linha que esta causando  o problema para poder-mos encontrar uma resolução para este caso."
 
 #: centreon-web/www/install/step_upgrade/step4.php:71
 #, php-format
 msgid "The SQL files are located in %s"
 msgstr "Os arquivos SQL estão localizados em %s"
 
+#: centreon-web/www/install/step_upgrade/step4.php:87
+#, php-format
+msgid "Please check the \"upgrade.log\" and the \"sql-error.log\" located in \"%s\" for more details"
+msgstr ""
```

Comment:
if you put it empty, br translation will display empty.
If you cannot translate it, just remove those lines